nubilanding/src/components/retiros/retiros.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from '../retiros/styles.module.scss'
```

Comment:
Están en la misma carpeta, así que podrías simplemente escribir:
```suggestion
import styles from './styles.module.scss';
```